packages/autocertifier/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@streamr/autocertifier",
+  "version": "0.0.1-tatum.6",
+  "description": "Automatic TSL Certificate Manager for Streamr Network",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/streamr-dev/network.git",
+    "directory": "packages/autocertifier"
+  },
+  "bin": {
+    "autocertifier": "dist/bin/run-auto-certifier.js"
+  },
+  "main": "dist/src/exports.js",
+  "types": "dist/src/exports.d.ts",
+  "license": "STREAMR NETWORK OPEN SOURCE LICENSE",
+  "author": "Streamr Network AG <contact@streamr.network>",
+  "scripts": {
+    "generate-protoc-code": "./proto.sh",
+    "build": "tsc -b tsconfig.node.json",
+    "check": "tsc -p ./tsconfig.jest.json --noEmit",
+    "clean": "jest --clearCache || true; rm -rf dist *.tsbuildinfo node_modules/.cache || true",
+    "eslint": "eslint --cache --cache-location=node_modules/.cache/.eslintcache/ '*/**/*.{js,ts}'",
+    "prepublishOnly": "npm run clean && NODE_ENV=production tsc -b tsconfig.node.json",
+    "test": "jest test/unit test/integration",
+    "test-unit": "jest test/unit",
+    "test-integration": "jest test/integration",
+    "test-production": "jest test/production",
```

```suggestion
    "test-end-to-end": "jest test/end-to-end",
```

We should convert the `production.test.ts` to an end-to-end test in which uses the test environment?